### PR TITLE
Numbers

### PR DESCRIPTION
**Pull request by dom-waterson**
Done all six task for the numbers slide

### CODE REVIEW

File: 2-numbers.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    };
+};
+
+checkNum(evenNum);
+checkNum(oddNum);
+
+//Task 2 - Convert Celsius into Farenheit
+var temp = 20;
+
+var convertCelToFah = function (temp) {
```
Comment (keithbarrow) [inline 2-numbers.js:20]: Don't fix this time, unless I make later comments - we use full explicit naming:
convertCelciusToFarenhiet

File: 2-numbers.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    return temp * 9 / 5 + 32;
+};
+
+console.log(convertCelToFah(temp));
+
+//Task 3 - Convert degrees into radians
+var degreeVal = 90;
+
+var convertDegToRad = function (degree) {
```
Comment (keithbarrow) [inline 2-numbers.js:29]: convertDegeneratesToRadicals :) again explicit naming - the teams are this strict.

File: 2-numbers.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    var radians = degree * Math.PI / 180
+    return radians;
+};
+
+console.log(convertDegToRad(degreeVal));
+
+//Task 4 - Given the base and perpendicular lengths of a right angle triangle, calculate the hypotenuse length
+var getHypotenuse = function (base, height) {
+    var hypotemuse = Math.sqrt(base * base + height + height);
+    return hypotemuse.toFixed(3);
```
Comment (keithbarrow) [inline 2-numbers.js]: This wasn't asked for - in real life you'd need to agree this as you are restricting the result with potential consequences . I expect you are doing this due to the floating point errors.

File: 2-numbers.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+var convertDegToRad = function (degree) {
+    var radians = degree * Math.PI / 180
+    return radians;
+};
+
+console.log(convertDegToRad(degreeVal));
+
+//Task 4 - Given the base and perpendicular lengths of a right angle triangle, calculate the hypotenuse length
+var getHypotenuse = function (base, height) {
+    var hypotemuse = Math.sqrt(base * base + height + height);
```
Comment (keithbarrow) [inline 2-numbers.js]: This has a mistake - also just squaring directly is probably neater.

File: 2-numbers.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    return hypotemuse.toFixed(3);
+};
+
+console.log("Hypotenuse is: " + getHypotenuse(3, 4));
+
+//Task 5 - Determine the length of a hypotenuse, for a given “opposite” length and angle
+var getHypot = function (oppLength, angle) {
```
Comment (keithbarrow) [inline 2-numbers.js:45]: get Hypothyriodism? etc etc etc

**Comment by dom-waterson** [comment]
fixed mistake for task 4

**Comment by keithbarrow** [comment]
:shipit:

**Pull request merged**